test/functional/discovered_hosts_controller_test.rb

```diff
@@ -122,18 +122,6 @@ def test_edit_form_submit_parameters
     assert_select '#host_hostgroup_id [selected]' do |e|
       assert_equal hostgroup.id.to_s, e.first[:value]
     end
-    assert_select '#host_architecture_id [selected]' do |e|
-      assert_equal hostgroup.architecture_id.to_s, e.first[:value]
-    end
-    assert_select '#host_operatingsystem_id [selected]' do |e|
-      assert_equal hostgroup.operatingsystem_id.to_s, e.first[:value]
-    end
-    assert_select '#host_medium_id [selected]' do |e|
-      assert_equal hostgroup.medium_id.to_s, e.first[:value]
-    end
-    assert_select '#host_ptable_id [selected]' do |e|
-      assert_equal hostgroup.ptable_id.to_s, e.first[:value]
-    end
```

Comment:
does that mean we can't test these as functional tests, only as integration? because of the new UI?

Reply:
yes, agree.